### PR TITLE
Wrote objective and residuals functions

### PR DESCRIPTION
**Pull request by RemingtonGerras**
Believe Christian and I have written the residual and objective functions. Untested, however...

### CODE REVIEW

**Comment by timothydmorton** [comment]
Referencing #7 #8

**Comment by timothydmorton** [comment]
@RemingtonGerras (and whomever you were working with)- accept the invitation for the usc-cosmolab group and I can officially assign you those issues.

File: lya/analysis.py [hunk 1]
```diff
@@ -22,11 +22,13 @@ def residuals(pars, data):
...
     """
 
+    return data.y - model(k, *pars)
 
 def objective(pars, data):
     """Objective function to miminize
     """
-
+    
+    return ((residuals(*pars, data)**2 / data.uncertainty**2).sum()
```
Comment (timothydmorton) [inline lya/analysis.py]: the `Data` object doesn't have an attribute called `.uncertainty`; it's called `.y_unc`

File: lya/analysis.py [hunk 2]
```diff
@@ -22,11 +22,13 @@ def residuals(pars, data):
         Data to be compared with model.
     """
 
+    return data.y - model(k, *pars)
```
Comment (timothydmorton) [inline lya/analysis.py]: What is `k` here?

**Review by timothydmorton — CHANGES_REQUESTED** [review]
See comments

File: lya/analysis.py (hunk 1 above)
Comment (syasini) [reply lya/analysis.py]: The current implementation of the data object has `data.k`, `data.P_k`, and `data.P_k_unc` attributes. Should we update the attributes here accordingly?

File: lya/.#analysis.py [hunk 3]
```diff
@@ -0,0 +1 @@
+remington@remington-Blade.23139:1580152665
```
Comment (timothydmorton) [inline lya/.#analysis.py]: please delete this file from the repo with `git rm lya/.#analysis.py --cached`

**Comment by timothydmorton** [comment]
@RemingtonGerras do you want to finish up this PR before the meeting today?

File: lya/.#analysis.py (hunk 3 above)
Comment (RemingtonGerras) [reply lya/.#analysis.py]: got it!

**Review by timothydmorton — APPROVED** [review]
Great, thanks!

**Pull request merged**